.golangci.yml

```diff
@@ -40,7 +36,6 @@ linters:
     - tenv
     - thelper
     - tparallel
-    - typecheck
```

Comment:
https://golangci-lint.run/welcome/faq/#why-do-you-have-typecheck-errors it can't be disabled or enabled as it's not really a linter and is always on afaict so no need to have it here